### PR TITLE
Business central fields api

### PR DESCRIPTION
**Pull request by ruuushhh**

### CODE REVIEW

File: tests/test_business_central/test_views.py [hunk 1]
```diff
@@ -25,3 +25,17 @@ def test_sync_dimensions(api_client, test_connection, mocker, create_temp_worksp
 
     response = json.loads(response.content)
     assert response['message'] == 'Business Central credentials not found / invalid in workspace'
+
+
+def test_sage300_fields(api_client, test_connection):
```
Comment (NileshPant1999) [inline tests/test_business_central/test_views.py:30]: central

File: tests/test_business_central/test_views.py [hunk 2]
```diff
@@ -25,3 +25,17 @@ def test_sync_dimensions(api_client, test_connection, mocker, create_temp_worksp
...
+
+    access_token = test_connection.access_token
+    url = reverse('business-central-fields', kwargs={'workspace_id': workspace_id})
+
+    api_client.credentials(HTTP_AUTHORIZATION='Bearer {}'.format(access_token))
+
+    response = api_client.get(url)
+    assert response.status_code == 200
+
+    assert response.data['results'] == []
```
Comment (NileshPant1999) [inline tests/test_business_central/test_views.py:41]: can you add something in the destination attribute and return some value here

File: tests/test_business_central/test_views.py (hunk 1 above)
Comment (ruuushhh) [reply tests/test_business_central/test_views.py:30]: fixed this here https://github.com/fylein/fyle-ms-business-central-api/pull/29

File: tests/test_business_central/test_views.py (hunk 2 above)
Comment (ruuushhh) [reply tests/test_business_central/test_views.py:41]: fixed this here https://github.com/fylein/fyle-ms-business-central-api/pull/29

Comment (NileshPant1999) [reply tests/test_business_central/test_views.py:41]: why fix it other pr
this is base pr no

**Review by NileshPant1999 — APPROVED** [review]

**Pull request merged**